transaction/core/src/blockchain/block.rs

```diff
@@ -10,7 +10,7 @@ use prost::Message;
 use serde::{Deserialize, Serialize};
 
 /// The current block format version.
-pub const BLOCK_VERSION: u32 = 0;
+pub const BLOCK_VERSION: u32 = 1;
```

Comment:
The old clients have to be upgraded, because we are doing a ledger format change, and the memos that we are adding affect the hash of the block (they feed into block contents and block id)

If we don't do this, those clients will still be broken because they will choke on the first block that has a memo in it, and say it doesn't pass validation

Reply:
Ugh, OK, we should definitely discuss this, because it's a gigantic shit-show that we hadn't planned on working on until Q4. We also need to look into how we're evolving things going forward. I am not super opposed to doing multiple ledger breaks, one now, one with the "real" evolution answer, but I'd like to keep it to a minimum.

Reply:
resolved in aug 9 meeting -- we decided there is a slight benefit to incrementing it